membership-attribute-service/app/configuration/Config.scala

```diff
@@ -3,17 +3,14 @@ package configuration
 import java.time.Duration
 
 import com.amazonaws.regions.Regions
-import com.amazonaws.services.sqs.AmazonSQSAsyncClient
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder
 import com.getsentry.raven.dsn.Dsn
-import com.github.dwhjames.awswrap.sqs.AmazonSQSScalaClient
```

Comment:
even though we are not using this library anymore here I could not remove it from the dependencies since it's still used in tests

Reply:
That library stops us receiving another deprecation warning about using a deprecated SQS Scala client... now that we've moved to a fully DI app this may not be necessary anymore.

Reply:
I removed the library and refactored all the test code that still needed it. That removed aws deprecation warning from tests as well